lib/fog/profitbricks/requests/compute/get_all_lans.rb

```diff
@@ -93,6 +93,8 @@ def get_all_lans(datacenter_id)
                         :method  => 'GET',
                         :path    => "/datacenters/#{datacenter_id}/lans?depth=1"
                     )
+                rescue => error
+                    Fog::Errors::NotFound.new(error)
```

Comment:
Hmm. I'm afraid this breaks the current behaviour. It might be wrong but as of now users are able to rescue each kind of error separately and with this new implementation they wont.
If you are going to change it later anyways, maybe we can leave it as is for now?